### DISCUSSION TITLE
The problem of not showing the table in the list of tables on the left in laravel voyager

### DISCUSSION THREAD

**Discussion by SOLAKA613** [thedevdojo/voyager#5344] [General]
When I want to create a table in a database using Laravel voyager, the table is created correctly but it does not appear in the list of tables on the left,Please any person help me with any idea,i use last version of laravel 8.

**Comment by emptynick** [comment]
Voyager won't show tables in the menu, but BREADs.
If you mean BREADs, make sure you set the permissions for that BREAD, otherwise it won't be shown.
